code/game/atoms.dm

```diff
@@ -243,23 +248,22 @@
 		return TRUE
 	return FALSE
 
-/atom/proc/get_examine_name(mob/user)
-	. = "\a [src]"
-	var/list/override = list(gender == PLURAL ? "some" : "a", " ", "[name]")
-	if(article)
-		. = "[article] [src]"
-		override[EXAMINE_POSITION_ARTICLE] = article
-	if(SEND_SIGNAL(src, COMSIG_ATOM_GET_EXAMINE_NAME, user, override) & COMPONENT_EXNAME_CHANGED)
-		. = override.Join("")
-
-/atom/proc/get_examine_string(mob/user, thats = FALSE)
-	. = "[icon2html(src, user)] [thats? "That's ":""][get_examine_name(user)]"
-
 /atom/proc/examine(mob/user)
-	to_chat(user, get_examine_string(user, TRUE))

```

Comment:
Also tiny nitpick here but what's the reason for this?

Reply:
it was part of the examine replacement text.

Which was part of the component.

Reply:
Unresolving since get_examine_string is a much cleaner way of handling this, and you didn't really bother with trying to replace any of the other uses of get_examine_string, meaning all of those other use cases are missing the ability to display blood-stained status